src/middlewared/middlewared/api/v25_10_0/acl.py

```diff
@@ -108,15 +134,37 @@ def check_inherit_only(self) -> Self:
 
 class NFS4ACE_BasicFlags(BaseModel):
     BASIC: NFS4ACE_BasicFlagset
+    """Basic inheritance behavior for NFS4 ACE.
+
+    * `INHERIT`: Apply to child files and directories
+    * `NOINHERIT`: Do not apply to child objects
+    """
 
 
 class NFS4ACE(BaseModel):
     tag: NFS4ACE_Tags
+    """Subject type for this ACE.
+
+    * `SPECIAL_OWNER`: File/directory owner
+    * `SPECIAL_GROUP`: File/directory primary group
+    * `SPECIAL_EVERYONE`: All users
+    * `USER`: Specific user account
+    * `GROUP`: Specific group
+    """
     type: NFS4ACE_EntryTypes
+    """Access control type.
+
+    * `ALLOW`: Grant the specified permissions
+    * `DENY`: Explicitly deny the specified permissions
+    """
     perms: NFS4ACE_AdvancedPerms | NFS4ACE_BasicPerms
+    """Permissions granted or denied by this ACE."""
     flags: NFS4ACE_AdvancedFlags | NFS4ACE_BasicFlags
+    """Inheritance and other behavioral flags for this ACE."""
     id: AceWhoId | None = None
+    """Numeric user or group ID when tag is `USER` or `GROUP`. `null` for special entries."""
```

Comment:
We need to specify that this is UID and GID so that user doesn't conflate with the datastore `id` value for user / group entries.